prisma/migrations/20221006181804_permissions/migration.sql

```diff
@@ -45,3 +45,40 @@ ALTER TABLE "_PrivilegeToUser" ADD CONSTRAINT "_PrivilegeToUser_A_fkey" FOREIGN
 
 -- AddForeignKey
 ALTER TABLE "_PrivilegeToUser" ADD CONSTRAINT "_PrivilegeToUser_B_fkey" FOREIGN KEY ("B") REFERENCES "User"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- Populate with Privileges
+
+INSERT INTO
+  "Privilege" ("id","nameEn", "nameFr", "descriptionEn", "descriptionFr", "permissions")
+VALUES
+  (gen_random_uuid(),'Base','Base','Base Permissions','Autorisations de Base','[
+    {"action":"create","subject":"FormRecord"},
+    {"action":["view","update","delete"],"subject":"FormRecord","conditions":{"users":{"$elemMatch":{"id":"${user.id}"}}}},
+    {"action":"update","subject":"FormRecord","fields":["publishingStatus"],"inverted":true},
```

Comment:
Conditions will have to be adjusted as we implement and test final permissions. `publishingStatus` is part of `formConfig` in this case so we can use `formConfig.publishingStatus`.

Reply:
100% Looking to refine these as we go.  Good catch on the sub property.  I'll modify.

Reply:
We'll also want to figure out what the 'subject' really is.  For example, are we really going to be checking a FormRecord the most or a FormConfiguration type?